README.md

```diff
@@ -29,7 +29,7 @@
 
 7. Execute start script to install all required packages
 
-    chmod +x start.sh && ./start.sh
+    chmod +x start.sh && . ./start.sh
```

Comment:
extra dot

Reply:
This is done to ensure that we can source the change in environment variables into the current shell